R/modeltests.R

```diff
@@ -140,15 +139,26 @@ startRuns <- function(test, model, mydir, gitPath, user) {
                     "startgroup=AMT titletag=AMT slurmConfig=\"--qos=standby --nodes=1 --tasks-per-node=12\" ",
                     "config/scenario_config.csv"))
 
+      # Create and save a list of runs that should have been started in order to determine later which runs were not started
       settings <- read.csv2("config/scenario_config.csv",
                              stringsAsFactors = FALSE,
                              row.names = 1,
                              comment.char = "#",
                              na.strings = "")
-
       runsToStart <- selectScenarios(settings = settings, interactive = FALSE, startgroup = "AMT")
       row.names(runsToStart) <- paste0(row.names(runsToStart), "-AMT")
       saveRDS(runsToStart, file = paste0(mydir, "/runsToStart.rds"))
+      
+      # start make test-full
+      # 1. pull changes to magpie develop
+      withr::with_dir("magpie", {
+            system(paste0(gitPath, " reset --hard origin/develop && ", gitPath, " pull"))
+      })
+      # 2. rename old test-full.log
+      system("mv test-full.log test-full-previous.log")
```

Comment:
You don't want to keep the logs long-term? I mean, you could also adjust `make test-full-slurm` to something like
```
sbatch --qos=priority --wrap="make test-full" --job-name=test-full --mail-type=END --output=test-full-2024-03-01.log --comment="test-full-2024-03-01.log"
```
in REMIND's makefile (not exactly sure how this would be implemented that the date is always adjusted). Or, at the end of the script, when everything is done, you copy the test-full.log to test-full-2024-03-01.txt and avoid like that that it is overridden.

Reply:
I wanted to avoid another file piling up in some folder that nobody needs. But if you really think we need the output of tests older than two weeks, I can change that.

Reply:
Can you just move it into `tests` subdirectory where it bothers no-one but is available if needed?